days/day-02/solutions/day02.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <cstring>
+#include <sstream>
+
+int main() {
+  const int MAXLINELENGTH = 30;
+  //Number of valid passwords
+  int valid_passwords = 0;
+  int acctually_valid_passwords = 0;
```

Comment:
Haha! Really really valid passwords? :+1: Good work on this one, thank you :smiley:

Reply:
Hehe, that is what happens when I want to solve part one first, then the second part "replaces" the correct answer in part one 😄 I'm honestly not that happy with how I read the input with stringline on line 18-20, but I failed when I tried to use sscanf like you did in your C solution. I'm expecting my solutions to become more and more janky 😄

Reply:
No problem :laughing: In the end, what matters is what is delivered to `stdout` :+1: The rest is just details :wink: